TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/ProxyClass.php

```diff
@@ -43,32 +45,32 @@ class ProxyClass
     protected $fullOriginalClassName;
 
     /**
-     * @var \TYPO3\Flow\Object\Proxy\ProxyConstructor
+     * @var ProxyConstructor
      */
     protected $constructor;
 
     /**
      * @var array
      */
-    protected $methods = array();
+    protected $methods = [];
 
     /**
      * @var array
      */
-    protected $constants = array();
+    protected $constants = [];
 
     /**
      * @var array
      */
-    protected $interfaces = array('\TYPO3\Flow\Object\Proxy\ProxyInterface');
+    protected $interfaces = ['\TYPO3\Flow\Object\Proxy\ProxyInterface'];
```

Comment:
Exactly like that it cannot be valid, as the backslash escapes the single quote… But in general that works, see https://3v4l.org/H6eAe